Framework/Algorithms/src/MultipleScatteringCylinderAbsorption.cpp

```diff
@@ -312,8 +314,8 @@ double calculate_msa_factor(const double radius, const double Q2,
  */
 void MultipleScatteringCylinderAbsorption::apply_msa_correction(
     const double angle_deg, const double radius, const double coeff1,
-    const double coeff2, const double coeff3, const vector<double> &wavelength,
-    vector<double> &y_val, std::vector<double> &errors) {
+    const double coeff2, const double coeff3, const HistogramX &wavelength,
+    HistogramY &y_val, HistogramE &errors) {
```

Comment:
@LamarMoore I think you overlooked this comment, could you clean this up as suggested (similar to how you did somewhere above, returning `Histogram` by value).